core/core.go

```diff
@@ -21,6 +21,10 @@ limitations under the License.
 // hardware/kernel/device specific programming implementation, if any.
 package core
 
+import (
+	"github.com/contiv/ofnet"
+)
```

Comment:
I'm not sure netplugin core depending on ofnet is a good idea.
Core plugin API is supposed to be driver independent.
So that we can replace OVS/openflow driver with other drivers like opflex, ipvlan etc in future.
We should create driver independent structure definitions for plugin layer to use.